src/Features/LanguageServer/Protocol/Registration/CompletionRegistrationOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.LanguageServer;
+
+internal class CompletionRegistrationOptions : CompletionOptions, ITextDocumentRegistrationOptions
```

Comment:
This is OK, but I'd really prefer updating MS.VS.LanguageServer.Protocol with these types here https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient

Reply:
Done